tests/unit/test_models.py

```diff
@@ -0,0 +1,23 @@
+from app.main.models import Movie
+
+# GIVEN a Movie model
+# WHEN a new movie entry is created
+# THEN check title, year, description, rating, review, img are defined correctly
+
+
+def test_new_movie():
```

Comment:
I don't like the idea of testing this implementation. 
What value give you this test? I think we should talk about test and of course I will give you materials about it.
 I think it will be better to continue adding tests in other MR